src/Commands/SiteCloneCommand.php

```diff
@@ -72,6 +72,10 @@ public function clonePantheonSite(
             $this->options[$key] = boolval( $value );
         }
 
+        if (!preg_match('/.*\..*/', $user_source) || !preg_match('/.*\..*/', $user_destination)) {
```

Comment:
I think you want `.+`s here rather than `.*`s.. 

`*` matches "zero or more" so it will happily match an empty string.  `.` is a valid `site.env` with this regex as-is.